modules/gapi/src/backends/streaming/gstreamingbackend.cpp

```diff
@@ -216,6 +253,21 @@ void cv::gimpl::BGR::Actor::run(cv::gimpl::GIslandExecutable::IInput  &in,
     out.post(std::move(out_arg));
 }
 
+cv::gapi::GKernelPackage cv::gapi::streaming::kernels()
+{
+    return cv::gapi::kernels<GOCVBGR>();
+}
+
+#else
+
+cv::gapi::GKernelPackage cv::gapi::streaming::kernels()
+{
+    // Still provide this symbol to avoid linking issues
+    util::throw_error(std::runtime_error("cv::gapi::streaming::kernels() isn't supported in standalone"));
+}
+
+#endif // !defined(GAPI_STANDALONE)
+
```

Comment:
Please add streaming backend sources to the list of source files which are compiled in standalone

Reply:
And you'll need to enclose <imgproc.hpp> include with standalone ifdefs

Reply:
Done